grapple/types/pages.py

```diff
@@ -117,22 +117,34 @@ class Meta:
         interfaces = (PageInterface,)
 
 
-def get_specific_page(id, slug, token, content_type=None):
+def get_specific_page(id, slug, url, token, content_type=None):
```

Comment:
I am not entirely sure about the method signatu here. https://github.com/torchbox/wagtail-grapple/blob/fbc2fdcd8f7bb48cdbeea005603b4a8ba8085b61/grapple/types/pages.py#L170 is explicit in passing everything as kwargs and any of these can be none...

Reply:
Fixed.